README.md

```diff
@@ -8,25 +8,24 @@ The docker images are available on [Amazon Corretto Official Images](https://hub
 
 To use Amazon Corretto Official Images from Docker hub run
```

Comment:
Include link to [dockerhub offical image ](https://hub.docker.com/_/amazoncorretto)

Reply:
The link is in line 7